com.unity.shadergraph/Documentation~/Upgrade-Guide-10-0-x.md

```diff
@@ -0,0 +1,64 @@
+# Upgrading to version 10.0.x of Shader Graph
+
+## Master Stack Graph Output
+
+Shader Graph has removed the Master Nodes and introduced a more flexible [Master Stack](Master-Stack) solution for graph output definition in 10.0. All graphs created in previous versions of Shader Graph can be opened and upgraded automatically in 10.0. See below for expected behavior and manual upgrade suggestions.
+
+### Automatic Upgrade from One Master Node to the Master Stack 
+If your graph only has one Master Node present, all of the data from that Master Node is automatically upgraded to the [Master Stack](Master-Stack) output. The following behavior is expected: 
+The correct [Target(s)](Graph-Target) is added to the Graph Settings tab of the [Graph Inspector](Internal-Inspector) and all settings from the Master Node cog menu to describe surface options are copied from the Master Node to the Target Settings. 
+A [Block](Block-Node) node is added to the [Master Stack](Master-Stack) for each port that existed on the Master Node. Any nodes that were connected to the Master Node ports are now connected to the corresponding [Block](Block-Node) Node. 
+Any values that were entered into the default value inputs of the ports on the Master Node are now copied to the corresponding [Block](Block-Node) Node. 
+The final shader is identical in appearance after upgrading. 
```

Comment:
This assumes there is a one to one correspondence, which with the old PBR node in HDRP I dont think there is?

Reply:
Answered below, but contradictory here then